app/uk/gov/hmrc/bindingtariffclassification/repository/SearchMapper.scala

```diff
@@ -69,13 +72,30 @@ class SearchMapper {
     }
   }
 
-  private def notEqualFilter(field: String): JsObject = {
-    Json.obj("$ne" -> field)
+  private def numberStartingWith(value: String): JsObject = {
+    Json.obj( regexFilter(s"^$value\\d*") )
+  }
+
+  private def contains(value: String): JsObject = {
+    Json.obj( regexFilter(s".*$value"), caseInsensitiveFilter )
```

Comment:
Do we not need .* at the end too?

Reply:
it works also without, but I can add it for more readability